### ISSUE TITLE
Unknown function get_networkRefs

### ISSUE THREAD

**Issue by nlamirault** [vmware/pyvcloud#4]
I try to creates a vApp using documentation. I've got an error on invalid function : 

```bash
$ ./test.py
Traceback (most recent call last):
  File "./test.py", line 167, in <module>
    '--fencemode': 'bridged'})
  File "/home/vagrant/Apps/pyvcloud/pyvcloud/vclouddirector.py", line 310, in create_vApp
    networkRefs = filter(lambda networkRef: networkRef.get_name().lower() == args["--network"].lower(), self.get_networkRefs())
AttributeError: 'VCD' object has no attribute 'get_networkRefs'

$ ag networkRefs *
README.rst
47:    networks = vcd.get_networkRefs()

pyvcloud/vclouddirector.py
86:    def get_networkRefs(self):
251:                    networkRefs = filter(lambda networkRef: networkRef.get_name().lower() == args["--network"].lower(), self.get_networkRefs())
252:                    if networkRefs:
253:                        network_name = networkRefs[0].get_name()
254:                        network_href = networkRefs[0].get_href()
```

**Comment by nlamirault** [comment]
I try this code 

```python
def get_networkRefs(self):
        vdc = self._get_vdc()
        resourceEntities = vdc.get_ResourceEntities().get_ResourceEntity()
        return [resourceEntity for resourceEntity in resourceEntities
                    if resourceEntity.get_type() == "application/vnd.vmware.vcloud.orgNetwork+xml"]

```

but no response : 

`̀``python
print "Networks:"
    for networks in vcd.get_networkRefs():
        print networks
```

**Comment by nlamirault** [comment]
I write this code : 

```python
def get_networkRefs(self):
        vdc = self._get_vdc()
        resourceEntities = vdc.get_AvailableNetworks().get_Network()
        print "----------------"
        print resourceEntities
        return [resourceEntity for resourceEntity in resourceEntities
                    if resourceEntity.get_type() == "application/vnd.vmware.vcloud.network+xml"]
```

but it fails later :

**Comment by pacogomez** [comment]
the code is available in the current master branch (v0.4, not yet tagged)

you will have to checkout from github instead of using pip repository, since it is not yet published there.

thanks

**Comment by nlamirault** [comment]
OK. I'm already use the git repository. Thanks
